### PR TITLE
Adds support for multiple SGs

### PR DESCRIPTION
**Pull request by danvaida**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -26,7 +26,7 @@ module "aws_vpc" {
 module "aws_elb" {
   enable = "${var.vpc_enable_bastion_host}"
 
-  source = "github.com/Flaconi/terraform-aws-elb?ref=v0.1.5"
+  source = "github.com/Flaconi/terraform-aws-elb?ref=v0.1.8"
```
Comment (cytopia) [inline main.tf:29]: @danvaida 
I suppose this should be `v0.1.9` as `v0.1.8` is already released and you're referring to the unreleaesd PR: https://github.com/Flaconi/terraform-aws-elb/pull/5

File: main.tf [hunk 2]
```diff
@@ -91,7 +92,7 @@ resource "aws_security_group" "bastion" {
     from_port       = "22"
     to_port         = "22"
     protocol        = "tcp"
-    security_groups = ["${module.aws_elb.security_group_id}"]
+    security_groups = ["${module.aws_elb.security_group_ids}"]
```
Comment (cytopia) [inline main.tf:95]: See comment for version change above. `0.1.8` only returns `security_group_id`, but not id**s**

**Review by cytopia — CHANGES_REQUESTED** [review]

File: main.tf (hunk 2 above)
Comment (danvaida) [reply main.tf:95]: Nice catch 🦅 👁️ 
But not enough. I used a moving tag instead:
Before:
Screenshot 2020-12-17 at 11 26 02

After:
Screenshot 2020-12-17 at 11 28 06

So the release tag can actually stay as-is.

File: main.tf (hunk 1 above)
Comment (danvaida) [reply main.tf:29]: Clarified [here](https://github.com/Flaconi/terraform-aws-vpc/pull/7/files#r544979801).

**Review by cytopia — APPROVED** [review]
:-)

**Pull request merged**